### ISSUE TITLE
Error thrown during migration when attempting to overwrite existing asset that doesn't exist yet

### ISSUE THREAD

**Issue by rgriffith** [hannonhill/Universal-Migration-Tool#19]
When running the tool with the option to overwrite existing assets, the deletion of the existing asset will fail with the following error and prevent the new asset from being created:

```
java.lang.Exception: Unable to identify an entity based on provided entity path 'foo/bar' and type 'page'
  at com.hannonhill.umt.service.RestApi.performApiRequest(Unknown Source)
  at com.hannonhill.umt.service.RestApi.performApiRequest(Unknown Source)
  at com.hannonhill.umt.service.RestApi.delete(Unknown Source)
  at com.hannonhill.umt.service.RestApi.createPage(Unknown Source)
  at com.hannonhill.umt.service.Migrator.createPages(Unknown Source)
  at com.hannonhill.umt.task.MigrationTask.run(Unknown Source)
```

### Expected Result

The error should be disregarded and the new asset should be created, instead of throwing an error and skipped.

## Related issues/tickets

- https://hannonhill.zendesk.com/agent/tickets/8022
- https://hannonhill.zendesk.com/agent/tickets/8657 gz#8657

**Comment by bradical** [comment]
Hi @rgriffith. Trying to assess priority here. Is this blocking the customer in any way?

**Comment by rgriffith** [comment]
It is blocking in a sense that it prevents the content from being migrated with the desired overwriting rule. The workaround is to choose a different rule to avoid the underlying problem, but it may be less than ideal depending on the customer's needs.

**Comment by bradical** [comment]
Priority level?

**Comment by bradical** [comment]
@rgriffith is there a priority on this? Did this come up for a customer or was discovered separately?

**Comment by rgriffith** [comment]
I would say this or next milestone. A customer encountered this when attempting to migrate their content, so I wouldn't be surprised if another customer ends up running into it soon.

**Comment by bradical** [comment]
and they are blocked or they found a workaround? if they found a workaround, I don't think it warrants `p2: next milestone`.

**Comment by tomusiaka** [comment]
I fixed this recently in #23 and the fix is in this release: https://github.com/hannonhill/Universal-Migration-Tool/releases/tag/vREST-API-1.0.3.
